WordPress/Classes/Services/MediaService.m

```diff
@@ -190,6 +192,48 @@ - (void)uploadMedia:(Media *)media
                 failure:failureBlock];
 }
 
+- (NSError *)processMediaUploadError:(NSError *)error {
```

Comment:
It's a nitpick but maybe `translateMediaUploadError` - process don't really declare what it does